### PR TITLE
Events controller

### PR DESCRIPTION
**Pull request by KojoSwiss**
- Events Controller
- Temporal form for new events creation

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  def index
+    @event = Event.all
+  end
+
+  def show
+  end
+
+  def new
+    @event = Event.new
+    @sports = Sport.all
```
Comment (ericblandet) [inline app/controllers/events_controller.rb:12]: not necessary, please delete

File: app/controllers/events_controller.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  def update
+    if @event.update(event_params)
+      redirect_to event_path
+    else
+      render 'edit'
+    end
+  end
+
+  def destroy
+    @event.destrop
```
Comment (ericblandet) [inline app/controllers/events_controller.rb:37]: typo : destroy

File: app/views/events/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<h1>Events#new</h1>
+<p>Find me in app/views/events/new.html.erb</p>
+
```
Comment (ericblandet) [inline app/views/events/new.html.erb:3]: add container?

File: app/views/events/update.html.erb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<h1>Events#update</h1>
```
Comment (ericblandet) [inline app/views/events/update.html.erb:1]: I think this view is not necessary

**Review by ericblandet — CHANGES_REQUESTED** [review]

**Review by Simon-Busch — APPROVED** [review]

**Comment by Simon-Busch** [comment]
Ok for merging

**Pull request merged**